tests/logging/test_opentracing.py

```diff
@@ -69,7 +71,7 @@ def test_start_active_span(self) -> None:
 
             # start_active_span should start and activate a span.
             scope = start_active_span("span", tracer=self._tracer)
-            span = scope.span
+            span = cast(jaeger_client.Span, scope.span)
```

Comment:
In this file we reference a bunch of properties which are specific to jaeger's implementation of `Span` so we cast from the generic `opentracing.Span` to `jaeger_client.Span`.

Reply:
Before I saw your comment, I drafted:

> Here and below: are the casts justified because we know we're in a logging context? I.e. so we don't have to worry about the sentinel span?

Your explanation shows my guess wasn't correct. Might it be worth a quick comment?

Reply:
Hopefully edab4ecf4384074e7eb41712028f0871bbf91fc6 makes sense?